changelog.md

```diff
@@ -1,8 +1,15 @@
 # Changelog
 
+## What's New (2/18/2020)
```

Comment:
You may work with Jianye for the complete change log. I think we will only release one that combines the change from you and Jianye.

Reply:
Ok. I will talk about Jianye offline.